crates/lib/src/bootc_composefs/boot.rs

```diff
@@ -55,6 +56,16 @@ pub(crate) const EFI_UUID_FILE: &str = "efiuuid.cfg";
 /// The EFI Linux directory
 const EFI_LINUX: &str = "EFI/Linux";
 
+/// Timeout for systemd-boot bootloader menu
+const SYSTEMD_TIMEOUT: &str = "timeout 5";
```

Comment:
This relates to https://github.com/coreos/bootupd/pull/978 - I think the default for systems-boot should actually be set at install time.